plane/source/docs/quadplane-vtol-tuning-process.rst

```diff
@@ -217,30 +217,50 @@ This test will allow to test the altitude controller and ensure the stability of
 
  If the QuadPlane in QHOVER starts to move up and down, the vertical position and velocity controllers may need to be reduced by 50%. These values are: :ref:`Q_P_POSZ_P <Q_P_POSZ_P>` and :ref:`Q_P_VELZ_P <Q_P_POSZ_P>`.
 
+.. note:: If the :ref:`Q_M_THST_HOVER<Q_M_THST_HOVER>` learned should be ~0.3-0.6. Higher values indicate that insufficient thrust is available, either due to motor system design, obstructed prop air flow by the fuselage or wings, or excessive yaw bias (see next section)
+
 Step 8: Yaw Bias
 ----------------
 
 A common problem in QuadPlanes is excessive amount of VTOL power being
-used to maintain yaw. This can be caused by:
+used to maintain yaw hold instead of providing lift. This can be caused by:
 
 - small misalignment of the VTOL motors
 - frame twist (often caused by wing twist) as thrust is applied
+- obstructed prop air flow by wings in :ref:`TVBS type tailsitters<tvbs>` making vectored yaw by motor tilt less effective. This can also cause enough reduction in normal thrust to make hovering and VTOL climb difficult.
 
 If too much power is needed to maintain yaw then the aircraft could
 lose yaw control during transitions, or could lose roll and pitch
-stability. For larger QuadPlanes it is common to need to deliberately
+stability. The most common symptom is a high hover throttle point, or even the inability to rise into a hover at high throttle stick position. For larger QuadPlanes it is common to need to deliberately
 tilt the motors by a couple of degrees to increase yaw authority.
 
+.. note:: for an X frame type (:ref:`Q_FRAME_TYPE<Q_FRAME_TYPE>` = 1), 
+    the motors should be tilted outwards. 
+    For an H frame (:ref:`Q_FRAME_TYPE<Q_FRAME_TYPE>` = 3) they should be tilted inwards.
+
 You should check the amount of thrust being used to maintain yaw by
 looking at the RATE YOut value in your hover logs. If it is over 10%
-(a value of 0.1) when hovering in no wind then you have a problem with
-yaw asymmetry that should be fixed.
+(a value of 0.1) when hovering in no wind with no pilot input,then you have a problem with
+yaw asymmetry that should be fixed. Just hovering in place may look fine 
```

Comment:
This is all a massive improvement.

I would put a paragraph break here though. Something like this.

> then you have a problem with yaw asymmetry that should be fixed.
>
> If you see no signs of yaw asymmetry, you should test what happens when you yaw left and right. Frames with insufficient yaw authority will show a dramatic rise in YOut for moderate yaw commands.

Reply:
this is subjective and not quantifiable...I think using RATE.Yout for anything than is already discussed is not as beneficial...I think I will leave it as is

Reply:
I was trying to make this last sentence less subjective. We start string with a good objective test of YOut in no wind and no pilot input, Then we say to add pilot input and "test this", but don't give any clarity of what we mean by that. Clearly we can't mean checking YOut>0.1, because we clearly stated that threshold is for no pilot input.

Reply:
Unless you meant "delete the proposed last sentence" when you said "leave as is", in which case, I think that's fine.

Reply:
made the hover RATE.Yout log check a recommended post first hover flight check for yaw bias